Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM thecodingmachine/php:7.4-v4-cli-node14
+FROM thecodingmachine/php:7.4.27-v4-apache-node16
```

Comment:
I wonder why you switched to the apache-version instead of the cli one?

Reply:
Thanks for spotting it! that was not intended